frontend/src/components/fragments/emotes/emote-submit-form.component.tsx

```diff
@@ -0,0 +1,91 @@
+import { bind } from 'decko';
+import React, { Component } from 'react';
+
```

Comment:
In this code, it uses state. To avoid having to type the state right now, I just cheated and used any again.  In the future, we'll want to make concrete interfaces for this too.